fastlane/Fastfile

```diff
@@ -112,16 +112,6 @@ def output_path
   return ENV.fetch('BITRISE_DEPLOY_DIR', "../build")
 end
 
-def branch_version
-  # https://rubular.com/r/A8FeXBzT1z4zJR
-  version_regex = /^releases\/v(\d+\.\d+(\.\d+)*)$/
-
-  if matches = git_branch.match(version_regex)
-    version = matches.captures[0]
-    puts "Version: #{version}"
-    return version
-  else
-    puts "No version number found in \"#{git_branch}\""
-  end
-  return nil
```

Comment:
Since we're open source, we won't use release branches for releases. Significant changes will bump the marketing_version within the app and we'll use that if a version isn't otherwise specified.